### DISCUSSION TITLE
Tap action on media player content (not only icon)

### DISCUSSION THREAD

**Discussion by lukaszwawrzyk** [Clooos/Bubble-Card#909] [Feature requests]
Hello,
Thanks for the great UI. I am migrating everything to bubble now.
I would like to be able to configure tap action for both icon and the rest of the card. It would be consistent with other cards, like button or select. 100% of their space is clickable. With music card, one has to remember to tap the icon. With mobile ui especially, it is just easier to tap anywhere. I usually setup the same action for icon and rest of the button to keep things easy.
Media player part is missing.

**Comment by Clooos** [comment]
Hi! Indeed I really need to add this option to more cards, like for the cover one as well. Maybe not for the next release but I keep this in mind!

**Reply by gandhiarpit** [reply]
Would love this with Climate card as well.
